tests/integration/tests/admin-areas/aggregates.test.ts

```diff
@@ -0,0 +1,56 @@
+import * as request from 'supertest';
+
+import { AdminLevel } from '../../helpers/API-service/enum/admin-level.enum';
+import { DisasterType } from '../../helpers/API-service/enum/disaster-type.enum';
+import { FloodsScenario } from '../../helpers/API-service/enum/mock-scenario.enum';
+import { getAccessToken, getServer, mock } from '../../helpers/utility.helper';
+
+export function getAdminAreaAggregates(
+  countryCodeISO3: string,
+  disasterType: DisasterType,
+  adminLevel: AdminLevel,
+  accessToken: string,
+): Promise<request.Response> {
+  return getServer()
+    .get(
+      `/admin-areas/aggregates/${countryCodeISO3}/${disasterType}/${adminLevel}`,
+    )
+    .set('Authorization', `Bearer ${accessToken}`);
+}
+
+export default function adminAreaAggregatesTests() {
+  const countryCodeISO3 = 'UGA';
+  const disasterType = DisasterType.Floods;
+  const adminLevel = AdminLevel.adminLevel3;
+  const scenario = FloodsScenario.Trigger;
+
+  describe('admin areas', () => {
+    let accessToken: string;
+
+    beforeAll(async () => {
+      accessToken = await getAccessToken();
+    });
+
+    it('should return list of admin area aggregates on GET', async () => {
```

Comment:
in principal same comment about scenarios as above, but at the same time maybe not as needed to duplicate all of that twice..